src/client.ts

```diff
@@ -698,15 +773,33 @@ export class VocdoniSDKClient {
   /**
    * Registers an account against vochain, so it can create new elections.
    *
-   * @param {{account: Account | null, faucetPackage: string | null}} options Additional
+   * @param {{account: Account | null, faucetPackage: string | null, sik: boolean | null}} options Additional
    * options, like extra information of the account, or the faucet package string
    * @returns {Promise<AccountData>}
    */
-  createAccount(options?: { account?: Account; faucetPackage?: string }): Promise<AccountData> {
+  createAccount(
+    options: { account?: Account; faucetPackage?: string; sik?: boolean } = {
+      account: null,
+      faucetPackage: null,
+      sik: true,
+    }
+  ): Promise<AccountData> {
```

Comment:
Same as I said before, careful with defaults and objects.

Reply:
This is a much better option for sure.

Can you open a new issue to tackle this in the whole Client?

I don't want to extend this PR much more, let's keep it only for anonymous stuff.

Reply:
Still, I think you should consider doing the merge in this method, since is the only one affected AFAIK, and it already gave some headache to other developers.